packages/sources/alpine/src/endpoint/tvl.ts

```diff
@@ -34,7 +34,8 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   const { network, vaultAddress } = validator.validated.data
 
   const rpcUrl = network.toUpperCase() == ETH ? config.ethereumRpcUrl : config.polygonRpcUrl
-  const provider = new ethers.providers.JsonRpcProvider(rpcUrl)
+  const chainId = network.toUpperCase() == ETH ? config.ethereumChainId : config.polygonChainId
```

Comment:
Is it really safe to assume that if the chain isn't ETH, it is automatically Polygon?  Seems like we should also check if the network == POLYGON and throw an error if it is not ETH or POLYGON.

Reply:
The ethers library itself will throw error if the chain is incorrect. Regarding the syntax i tried to use the same that is done for rpcUrl